packages/physics-physx/src/PhysXPhysicsManager.ts

```diff
@@ -0,0 +1,230 @@
+import { PhysXPhysics } from "./PhysXPhysics";
+import { Ray, Vector3 } from "@oasis-engine/math";
+import { IPhysicsManager } from "@oasis-engine/design";
+import { PhysXCollider } from "./PhysXCollider";
+
+/** Filtering flags for scene queries. */
+export enum QueryFlag {
+  STATIC = 1 << 0,
+  DYNAMIC = 1 << 1,
+  ANY_HIT = 1 << 4,
+  NO_BLOCK = 1 << 5
+}
+
+/**
+ * Physics state
+ */
+export enum PhysicsState {
```

Comment:
No need to `export` for use in the file,.
Belongs to auxiliary classes and auxiliary enumerations，You should put them after main class, not before.

Reply:
done